src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala

```diff
@@ -90,10 +90,12 @@ class CassandraJournal(cfg: Config) extends AsyncWriteJournal with CassandraReco
   }
 
   private var sessionUsed = false
+  private val metricsCategory = s"cassandra.journal.${context.system.name}.${self.path.name}"
```

Comment:
we should use the plugin config path as the key to the metrics, that is `self.path.name`, that will be unique within the actor system